modules/vstudio/vs2005_fsproj.lua

```diff
@@ -0,0 +1,86 @@
+--
+-- vs2005_fsproj.lua
+-- Generate a Visual Studio 2005-2010 F# project.
+-- Copyright (c) 2009-2014 Jason Perkins and the Premake project
```

Comment:
First line should probably be the full path, but the other files above don't seem to do this like the test files do. Second line should probably not have a version range? 2005+ perhaps? Third line should be 2017-2017.